docs/topics/assembly_interacting-with-the-secured-booster.adoc

```diff
@@ -0,0 +1,20 @@
+// This assembly uses a parameterized module. Parameters used:
+//
+//   parameter-runtime: A segment of the endpoint name
+//
+// Rationale: The API endpoint procedure is identical on all runtimes
+
+[#interacting-with-the-secured-booster]
+= Interacting with the {name-mission-secured} Booster
+
+The Secured booster provides a default HTTP endpoint that accepts `GET` requests if the caller is authenticated and authorized.
```

Comment:
Should the beginning of this sentence also use `{name-mission-secured}`? (I don't know myself.)

Reply:
Thanks, good catch, it should.